system_baseline/models.py

```diff
@@ -106,23 +110,22 @@ class SystemBaselineMappedSystem(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     account = db.Column(db.String(10), nullable=False)
+    org_id = db.Column(db.String(36))
     system_baseline_id = db.Column(
         UUID(as_uuid=True), ForeignKey("system_baselines.id"), nullable=False
     )
     system_id = db.Column(UUID(as_uuid=True), nullable=False, index=True)
 
     @classmethod
-    def delete_by_system_ids(cls, system_ids, account_number):
-        cls.query.filter(cls.system_id.in_(system_ids), cls.account == account_number).delete(
-            synchronize_session="fetch"
-        )
+    def delete_by_system_ids(cls, system_ids, account_number, org_id):
```

Comment:
you added org_id into the values passed in but didn't use it; is this just a precursor to swapping out account_number later or did you mean to do more within the function here?

Reply:
You are right, I missed this. Thank you.